common/rpc/interceptor/telemetry.go

```diff
@@ -62,19 +62,24 @@ var (
 
 // static variables used to emit action metrics.
 var (
-	respondWorkflowTaskCompleted = "RespondWorkflowTaskCompleted"
+	respondWorkflowTaskCompleted = "RespondWorkflowTaskCompletedRespondWorkflowTaskCompleted"
 	pollActivityTaskQueue        = "PollActivityTaskQueue"
 
 	grpcActions = map[string]struct{}{
-		metrics.FrontendQueryWorkflowScope:                    {},
-		metrics.FrontendRecordActivityTaskHeartbeatScope:      {},
-		metrics.FrontendRecordActivityTaskHeartbeatByIdScope:  {},
-		metrics.FrontendResetWorkflowExecutionScope:           {},
-		metrics.FrontendStartWorkflowExecutionScope:           {},
-		metrics.FrontendSignalWorkflowExecutionScope:          {},
-		metrics.FrontendSignalWithStartWorkflowExecutionScope: {},
-		metrics.FrontendRespondWorkflowTaskCompletedScope:     {},
-		metrics.FrontendPollActivityTaskQueueScope:            {},
+		"QueryWorkflow":                    {},
+		"RecordActivityTaskHeartbeat":      {},
+		"RecordActivityTaskHeartbeatById":  {},
+		"ResetWorkflowExecution":           {},
+		"StartWorkflowExecution":           {},
+		"SignalWorkflowExecution":          {},
+		"SignalWithStartWorkflowExecution": {},
+		"RespondWorkflowTaskCompleted":     {},
+		"PollActivityTaskQueue":            {},
+		"CreateSchedule":                   {},
+		"UpdateSchedule":                   {},
+		"DeleteSchedule":                   {},
+		"PatchSchedule":                    {},
+		"UpdateWorkflowExecution":          {},
```

Comment:
What's the benefit of inlining the strings; couldn't we just use the `const`s? The benefit of that is they have (some) documentation.

Reply:
I guess it is easer to read with just the string literal, but maybe not. The old metrics.FrontendXxx const was used to tag metrics for each method (not related to actions here) before we moved that tag into auto gen. Now they are not used anywhere else, so I think it is better to remove them. If we want const, we can define private const here in this file.